Makefile

```diff
@@ -39,6 +39,9 @@ cross-clippy:
 	# Cross compile for pi, targeting real hardware
 	CROSS_CONTAINER_OPTS="--platform linux/amd64" cross clippy --bin piggui --release --features "gui","pi_hw" --tests --no-deps --target=aarch64-unknown-linux-gnu
 	CROSS_CONTAINER_OPTS="--platform linux/amd64" cross clippy --bin piglet --release --features "pi_hw" --tests --no-deps --target=aarch64-unknown-linux-gnu
+	# Cross compile for pi3b, targeting real hardware
```

Comment:
You could skip this, as clippy lifting should not depend on the arch (at least if we never add anything different between Pi3B and other devices/arch), it will just make the cross-clippy target slower for the same results (repeated).